source/usage-examples/changestream.txt

```diff
@@ -1,9 +1,9 @@
 .. _golang-watch:
 .. _golang-usageex-monitor-changes:
 
-====================
-Monitor Data Changes
-====================
+============================
+Monitor Data Changes Example
```

Comment:
```suggestion
Open a Change Stream Example
```
S: Discussed this title with Sarah. I would change this title to include Change Stream, as the page titles for pages within the Usage Example section trend towards being more product or method name focused. "Open a Change Stream" will do to ensure page title meets length requirements.